### DISCUSSION TITLE
Webpage not centered

### DISCUSSION THREAD

**Discussion by ttmms** [Igalia/cog#407] [Q&A] [answered]
Hi,

I'm running `cog --platform=wl localhost` and the website is off-center. The screen is in portrait mode, but it feels like the website is in landscape? Chromium shows webpages correctly and weston itself is centered. Any ideas what setting could be wrong?

Thanks!

**Comment by ttmms** [comment, marked as answer]
Answering myself, had to set environment variable COG_PLATFORM_**WL**_VIEW_FULLSCREEN=1

Not COG_PLATFORM_**FDO**_VIEW_FULLSCREEN=1. 

`--platform=fdo` is deprecated but works, just gives a warning.
But COG_PLATFORM_FDO_VIEW_FULLSCREEN=1 does not even work

**Comment by aperezdc** [comment]
I suppose we could re-add checking the old environment variables and printing a warning if they are found in the environment, but I would rather avoid that. For now I have posted https://github.com/Igalia/cog/pull/408 to document the variables, which is something we were missing.
